src/python/requirements.txt

```diff
@@ -3,9 +3,8 @@ argcomplete>=1.9.4,<2.0.0; python_version < "3.10"
 websocket-client==0.54.0
 python-dateutil>=2.5
 psutil>=5.9.3
-requests>=2.8.0,<2.28; python_version < "3.7"
-requests>=2.8.0,<2.29; python_version >= "3.7"
-cryptography>=3.4.2,<41; python_version > "3.5"
+certifi
+urllib3>=1.26.18,<2.1.0
```

Comment:
why don't we pin the most recent version? is it because of SecureTransport TLS deprecation?

Reply:
I have not done extensive testing with `2.1.0` which has many deprecations, but seems okay so far in testing. `urllib3` deprecations on new minor version updates have caused dxpy errors in the past. 
`v1.26.x` is still maintained and `<2` could be required by other packages a user may have installed.